config.json

```diff
@@ -61,6 +61,13 @@
       ]
     },
     {
+      "slug": "acronym",
+      "difficulty": 2,
+      "topics": [
+        "String Manipulation"
```

Comment:
> based on student feedback.

It's a good idea. https://github.com/exercism/discussions/issues/92 and https://github.com/exercism/discussions/issues/86 might have some useful information.

> 1. In the submission page, the User finds an option to rate the difficulty of the exercise; or
> 1. Upon archiving the exercise, the User is prompted with a request to rate the difficulty exercise.

> I favour option **1** due to many Users (including me) not archiving their exercises.

Same. There are many I don't archive because I deem that feedback would always be welcome on it.

> A Student already familiar with the Language could rate an exercise Very Easy, while beginners would rate it Hard

We may be able to adjust for it if we let students declare familiarity with the language: 

* https://github.com/exercism/exercism.io/issues/2164
* https://github.com/exercism/exercism.io/issues/2594
* https://github.com/exercism/discussions/issues/16

> too many exercises on that topic, causing students to get bored.

I see it is a track-specific version of https://github.com/exercism/x-common/issues/326

> I apologise for often giving such big comments/replies

One could argue that my response is either too short and doesn't give the consideration deserved, or that it is too long if one includes all the text in everything I linked to as part of my answer.

I think the suggestions are good, so there is no need to apologise. And the links should show there are others interested in the same, so there are places to add your voice and get more people interested. These would benefit all language tracks of Exercism!